### DISCUSSION TITLE
For some unknown reasons the codeaction bindings were gone in the lastest update

### DISCUSSION THREAD

**Discussion by directormac** [LazyVim/LazyVim#1039] [Q&A] [answered]
Is it just me?

**Comment by folke** [comment, marked as answer]
fixed!

**Comment by directormac** [comment]
Offf Nevermind @folke  beat me again.  man you are awesome!
![image](https://github.com/LazyVim/LazyVim/assets/5866196/f9d49c4b-1cb5-4a35-9781-d9605c76e250)
